fees/kairos/index.ts

```diff
@@ -0,0 +1,56 @@
+import { FetchOptions, SimpleAdapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+
+import { getETHReceived } from "../../helpers/token";
+
+// 0x64a0ddF7469d52828a026b98A76F194637DaAd2C(ExpressLanAuction Contract)
+
+// https://docs.kairos-timeboost.xyz/submission-api
+const WETH_ADDRESS = '0x82aF49447D8a07e3bd95BD0d56f35241523fBab1';
+const KAIROS_PAYMENT_ADDRESS = '0x60E6a31591392f926e627ED871e670C3e81f1AB8';
+const KAIROS_AUCTION_BIDDER_ADDRESS = '0x2b38a73dd32a2eafe849825a4b515ae5187eda42';
+
+const AUCTIONRESOLVED_EVENT_ABI = 'event AuctionResolved(bool indexed isMultiBidAuction, uint64 round, address indexed firstPriceBidder, address indexed firstPriceExpressLaneController, uint256 firstPriceAmount, uint256 price, uint64 roundStartTimestamp, uint64 roundEndTimestamp)'
+
+const fetchFees = async (_a: any, _b: any, options: FetchOptions) => {
+    const dailyFees = options.createBalances();
+    const dailyCost = options.createBalances();
+
+    const logs = await options.getLogs({
```

Comment:
you can add a topics filter field here, this you fetch only the interested logs, downside of that is, we dont use indexer (multi topic filter is not supported atm iirc)